### ISSUE TITLE
Word nodes 11325-11326: אֲרַ֥ם נַֽהֲרַ֖יִם

### ISSUE THREAD

**Issue by sethbam9** [ETCBC/bhsa#9]
In the 2021 dataset (unlike the 2017 data), Mesopotamia (Strong's H0763), אֲרַ֥ם נַֽהֲרַ֖יִם, occurs as two separate nodes with different lexical values: 11325 'אֲרַם' and 11326 'נַהֲר' (river). I think they should be a single node with a single lexical value. 

`print(F.g_lex_utf8.v(11325), F.g_lex_utf8.v(11326))`

**Comment by PeursenWTvan** [comment]
The data encoding is based, as much as possible on linguistic observations, rather than encyclopedic information or Strong’s numbers. In this case there is a construct state and a dual, to keep thoses features retrievable, and also be able to handle constructions such as פַּדֶּ֣נָֽה אֲרָ֔ם (with the so called locative element attached to the first element of the alleged compound name) we annotate them separately. In this way we postpone the conclusion as to when these word combinations became lexicalized as one proper noun.

**Comment by sethbam9** [comment]
@PeursenWTvan Thank you for the feedback. Are you aware if there's any work being done, or that has been done to link the ETCBC Hebrew data to Strong's numbers?

**Comment by PeursenWTvan** [comment]
Yes, I remember such work has been done, but I don't remember the project. In fact, although it many people may be helpful, adding such numbers runs counter to the spirit in which the ETCBC database was created. What you can do is ask around in the ETCBC slack community on etcbc-vu.slack.com.

**Comment by sethbam9** [comment]
Do you know how I'd go about joining that slack community? It appears to require a certain email extension to join.
